diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index fb5a49ac44c40f627b24671dfb1dfabbc52d65c3..0000000000000000000000000000000000000000
diff --git a/src/dependency-version-checker.js b/src/dependency-version-checker.js
index 05215f3a321bbbb907f24a67498da7689c3bd1e0..5b55c9c5c0c6708786a05345a2b79a7f07f2a89e 100644
--- a/src/dependency-version-checker.js
+++ b/src/dependency-version-checker.js
@@ -34,7 +34,12 @@ class DependencyVersionChecker {
       // to only resolvePackagePath once.
       let addon = this._parent._addon;
       let basedir = addon.root || getProject(addon).root;
+
       this._jsonPath = resolvePackagePath(this.name, basedir);
+
+      if(this._jsonPath === null) {
+        console.log('JSON_PATH IS NULL for', this.name, basedir);
+      }
     }
 
     if (this._version === undefined && this._jsonPath) {
